### ISSUE TITLE
Vue files not linted using mithotyn

### ISSUE THREAD

**Issue by NicoAiko** [fimbullinter/wotan#601]
I think the problem is obvious just by reading the title.

As Vetur fails to actually work, I switched to wotan.
But as it seems, Wotan is not able to actually lint Vue files in VSCode.

I checked the regular `.ts` (TypeScript) files and they work just fine.

I also use `valtyr` and `ve`.

As an example, here's my `.fimbullinter.yaml`

```yaml
# Tell wotan to use the valtyr module for linting.
modules:
  - "@fimbul/valtyr"
# Valtýr configuration.
valtyr:
  overrides:
    # Use the Vé preprocessor for all *.vue files.
    # Valtýr will still run for *.ts files by default.
    - files: "*.vue"
      processor: "@fimbul/ve"
exclude:
  - "**/*.d.ts"
```

In my `tsconfig.json` file, I also added the `mithotyn` plugin like so:

```json
"plugins": [
      { "name": "@fimbul/mithotyn", "displayErrorsAsWarnings": true }
    ]
```

Is the `mithotyn` plugin not capable to lint vue files?
If so, wotan is no solution for me...

**Comment by ajafff** [comment]
This is kinda expected. The docs mention the following:

> There are some limitations of the current implementation.
> * doesn't use processors at all

Which means it doesn't use `processor: "@fimbul/ve"` to make `.vue` files available to TypeScript.

However, I would expect the plugin that powers VSCode's language service for Vue files to do that kind of preprocessing and also respect the `plugins` option in your `tsconfig.json`.
It seems Vetur doesn't load TypeScript LanguageService plugins at all. Otherwise `@fimbul/mithotyn` or `typescript-tslint-plugin` would work when configured as `plugins` in `tsconfig.json`.
Related issue: https://github.com/vuejs/vetur/issues/170 (but you probably already know that one).

TL;DR there's not much this project can do about that (without duplicating a lot of code from Vetur).

**Comment by NicoAiko** [comment]
I see. Thanks for clearing up!
